docs/src/main/sphinx/connector.md

```diff
@@ -43,3 +43,12 @@ Thrift          <connector/thrift>
 TPCDS           <connector/tpcds>
 TPCH            <connector/tpch>
 ```
+
+Additional features for object storage connectors:
+
+```{toctree}
```

Comment:
We could hide this toctree so the file does not show up in the left hand nav. Or we could leave it active and move more object storage pages into here .. such as the file formats and metastores now.

Reply:
Will use this in filesystem PR as well .. and probably leave hidden as discussed with @electrum